<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Evaluation</class>
 <widget class="QDialog" name="Evaluation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="evaluationLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Evaluate your plate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="commentTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="note1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Awful</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="note2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bad</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="note3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="note4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Good</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="note5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Perfect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
